wp-content/plugins/core/src/Menus/Menu_Order.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Plugin\Menus;
+
+class Menu_Order {
```

Comment:
I would make sure to prefix your work with `Admin_` as this `menu` namespace is often used for frontend menus as well.